core/comm/producer.go

```diff
@@ -14,10 +14,13 @@ import (
 	"time"
 
 	"github.com/hyperledger/fabric/common/flogging"
+
 	"google.golang.org/grpc"
 )
 
-var logger = flogging.MustGetLogger("ConnProducer")
+var (
+	connProdLogger = flogging.MustGetLogger("ConnProducer")
```

Comment:
We use lowercase loggers everywhere else.

Reply:
Well but renaming the logger is not the purpose of this pull request, I am just changing the variable name

Reply:
I left it as is, I just renamed the variable name.

I removed the file from the change set. 

Feel free to change the logger name in a separate pull request if you feel this is needed